app/components/FactsPageContent.tsx

```diff
@@ -1,19 +1,29 @@
+'use client';
+
 import { Earthquakes } from '@/types';
 import StatsSection from './StatsSection/StatsSection';
+import useSyncAtom from '@/store/useSyncAtom';
+import { allDailyEventsAtom, dailyEventsWithTimesAtom } from '@/store';
 
 type FactsPageContent = {
+  allDailyEvents: any; // this is ALL the earthquakes for the day, regardless of magnitude
+  dailyEventsWithTimes: any; // those of significant magnitude with times they occured
   topMagnitudeEvents: Earthquakes;
   totalCount: number;
   weeklyTotalCount: number;
   weeklyTopMagnitudeEvents: Earthquakes;
 };
```

Comment:
Same note as on the atoms - this can be heavily consolidated, and a lot less confusing as a result.